include/seastar/testing/perf_tests.hh

```diff
@@ -45,22 +45,25 @@ public:
     uint64_t allocations = 0;
     uint64_t tasks_executed = 0;
```

Comment:
Annoying that we get 3-4 IPC in benchmarks but <1 in real life.

Reply:
agreed